lib/app/data/models/alarm_model.dart

```diff
@@ -162,6 +165,7 @@ class AlarmModel {
     shakeTimes = alarmData['shakeTimes'];
     label = alarmData['label'];
     isOneTime = alarmData['isOneTime'];
+    ringtoneName = alarmData['ringtoneName'];
```

Comment:
I just wanted to have a relation between `ringtoneName`, and `isarId`. Because the unique ID assigned by Isar is autoincremented, and you can't find the isarId directly, through hashing, I can directly get the isarId by just ringtoneName.

Yes, in this implementation, every file has to have a unique name to be able to be added in. 

One thing that can be done is that we can assign the unique ID by Isar directly, but then we'll have to fetch the ringtoneModel just by ringtoneName, and not uid.

Reply:
Ah yes, I guess the conflict in the documents directory is a good argument to move ahead with ringtoneName. There's an obvious way to rename the file if its conflicting but I think we can add that in the future.